priv/styles.css

```diff
@@ -120,10 +121,24 @@ h1, h2, h3, h4, h5, h6 {
   margin-bottom: var(--gap-l);
 }
 
-.package-list h2 {
+.package-list li div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+  gap: var(--gap);
+
   margin: 0 var(--gap) var(--gap-s) 0;
 }
 
+.package-list li div span {
```

Comment:
Let's add classes to these elements rather than using the type of the element. Make it a bit clearer and avoid accidentally applying this to other in future.

Reply:
Good point. Done!